### PR TITLE
Benchmarks: Build Pipeline - add AMF in third party and build AMF encoding latency test

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
add AMF in third party and build AMF encoding latency test.

### CODE REVIEW

**Review by cp5555 — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -21,3 +21,6 @@
 [submodule "third_party/gpu-burn"]
 	path = third_party/gpu-burn
 	url = https://github.com/wilicc/gpu-burn.git
+[submodule "third_party/AMF"]
+	path = third_party/AMF
+	url = https://github.com/GPUOpen-LibrariesAndSDKs/AMF.git
```
Comment (abuccts) [inline .gitmodules:26]: the module seems too large (GB level) and is not used in most cases
maybe move to dockerfile or makefile and clone only when building directx?

Comment (yukirora) [reply .gitmodules]: moved to the third_party/Makefile directly

File: .github/workflows/build-image.yml [hunk 2]
```diff
@@ -64,6 +64,17 @@ jobs:
           sudo apt-get clean
           sudo docker rmi $(sudo docker images --format "{{.Repository}}:{{.Tag}}" --filter=reference="node" --filter=reference="buildpack-deps")
           df -h
+      - name: Free Up GitHub Actions Ubuntu Runner Disk Space 🔧
```
Comment (abuccts) [inline .github/workflows/build-image.yml:67]: don't need any more?

Comment (yukirora) [reply .github/workflows/build-image.yml]: ok, removed

**Review by abuccts — APPROVED** [review]

**Pull request merged**